index.html

```diff
@@ -256,6 +260,7 @@ <h3 id="out-of-scope">Out of Scope</h3>
               <li>Browser APIs</li>
               <li>Specific Implementations of DID Methods and Protocols</li>
```

Comment:
Could change this to "Specific DID Method specifications or Protocol specifications"